composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "utopia-php/fetch",
+    "description": "A simple library that provides an interface for making HTTP Requests.",
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "php": ">=8.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "laravel/pint": "^1.5.0",
+        "phpstan/phpstan": "^1.10"
+    },
+    "scripts": {
+        "lint": "./vendor/bin/pint --test --config pint.json",
+        "format": "./vendor/bin/pint --config pint.json",
+        "check": "./vendor/bin/phpstan analyse -c phpstan.neon",
+        "test": "./vendor/bin/phpunit --configuration phpunit.xml --debug"
```

Comment:
Can you please run

```sh
composer check
composer test
composer format
```
and post their outputs as part of the PR ? I think for some reason github blocks actions from external contributors the first time 🤔

Reply:
sure